### ISSUE TITLE
Clean up format for atomic scan --detail

### ISSUE THREAD

**Issue by baude** [projectatomic/atomic#156]
Once PR 152 is merged, the report created with the --detail switch should be slightly cleaned up and tweaked for better formatting.

**Comment by rhatdan** [comment]
It is merged.

**Comment by rhatdan** [comment]
Any update on this, Can I close?

**Comment by baude** [comment]
Once PR #164 is merged, I'll have another PR to take care of this.  It is now being combined with the input parsing we talked about for sifting/filtering image/container names.

**Comment by rhatdan** [comment]
#164 is merged
